catalyst_voices/apps/voices/lib/pages/treasury/treasury_page.dart

```diff
@@ -32,31 +28,45 @@ class TreasuryPage extends StatefulWidget {
 }
 
 class _TreasuryPageState extends State<TreasuryPage> {
+  late final SectionsController sectionsController;
+
+  @override
+  void initState() {
+    sectionsController = SectionsController();
+
+    _populateSections();
+
+    super.initState();
```

Comment:
a bit unusual to put super.initState() as the last call, was there a reason for it?